src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,30 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { TodowithUser } from '../types';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
```

Comment:
```suggestion
interface Props = {
```
please, prefer the interface